app/models/cache/JourneyType.scala

```diff
@@ -21,7 +21,7 @@ import play.api.libs.json.{Format, Reads, Writes}
 object JourneyType extends Enumeration {
   type JourneyType = Value
 
-  val ARRIVE, DEPART, ASSOCIATE_UCR, DISSOCIATE_UCR, SHUT_MUCR = Value
+  val ARRIVE, DEPART, ASSOCIATE_UCR, DISSOCIATE_UCR, SHUT_MUCR, VIEW_SUBMISSIONS = Value
```

Comment:
I'm not sure about approach, I think we should handle submission in slightly different way, without Answers, without JourneyType, just Choice